recipes/cf_xarray/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cf_xarray" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 960562cde23cc460d478b936d5967beed7bbfa200bef2c4ab4c5ba24c132c838
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - xarray
+    - python >=3.6
+
+test:
+  imports:
+    - cf_xarray
+  requires:
+    - pytest
```

Comment:
If you are not running the tests you don't need this but don't worry, we can remove it later.